tests/topotests/bgp_vpnv4_noretain/test_bgp_vpnv4_noretain.py

```diff
@@ -169,37 +181,280 @@ def test_bgp_no_retain():
     assert result is None, assertmsg
 
 
-def test_bgp_retain():
+def test_bgp_no_retain_step1():
+    """
+    Check bgp no retain route-target all on r1
+    """
+
+    rname = "r1"
+    check_show_bgp_ipv4_vpn(rname, "ipv4_vpn_routes_no_retain_init.json")
```

Comment:
Just making certain we don't need to have some sort of loop or multiple checks in case bgp doesn't converge as fast as the script gets to this point ... there are a couple of other of these throughout